src/Message/PurchaseRequest.php

```diff
@@ -112,4 +114,87 @@ private function resolvePageAction()
 
         return 'PAYMENT';
     }
+
+    public function getPaymentConfig()
+    {
+        return $this->getParameter('paymentConfig');
+    }
+
+    /**
+     * @param string $type
+     * @param array $values
+     * @return PurchaseRequest
+     * @throws InvalidArgumentException
+     * @throws InvalidRequestException
+     */
+    public function setPaymentConfig($type, $values = [])
+    {
+        if (!in_array($type, ['SINGLE', 'MULTI', 'MULTI_EXT'])) {
+            throw new InvalidArgumentException('type is not allowed');
+        }
+
+        $value = 'SINGLE';
+
+        switch ($type) {
+            case 'MULTI':
+                if (!isset($values['first'], $values['count'], $values['period'])) {
+                    throw new InvalidArgumentException('Invalid parameter for type MULTI');
+                }
+
+                $values['first'] = $this->formatPaymentConfigAmount($values['first']);
+                $value = $type . ':' . http_build_query($values, '', ',');
```

Comment:
in the doc it's said that it should be `;` for both type (https://payzen.io/fr-FR/form-payment/reference/vads-payment-config.html)

Reply:
I missed this one, thanks :')